setup.py

```diff
@@ -23,8 +23,8 @@
     packages=['daceml'],
     package_data={'': ['*.cpp']},
     install_requires=[
-        'dace@git+https://github.com/spcl/dace.git@d3571a8', 'onnx == 1.7.0',
-        'torch'
+        'dace@git+https://github.com/orausch/dace.git@daceml_branch',
```

Comment:
sad. This is only because of the serialization issue?

Reply:
Yes, at the moment that's the blocker